apollo-federation/src/merger/merge_directive.rs

```diff
@@ -180,17 +178,30 @@ impl Merger {
         {
             self.merge_custom_core_directive(name)?;
         } else {
-            let sources = self.get_sources_for_directive(name)?;
```

Comment:
do we need the `get_sources_for_directive`? on a brief search it looks like it is not used anywhere else?